ampadmin.yml

```diff
@@ -0,0 +1,49 @@
+version: "3"
+
+services:
+
+    server:
+        image: ${imageAmp}
+        command: ["adm-server"]
+        deploy:
+            mode: replicated
+            replicas: 1
+            labels:
+                io.amp.role: "infrastructure"
+                io.amp.mapping: "ampadm:31315:tcp:8081"
```

Comment:
we should only consider 80 and 443 as normal ports. 8080 is barely OK, because wildly used, but there's no guarantee it will work for everybody. We can try with ports > 2^15, but I'm not 100% sure it's a great idea. Best solution would have been virtual hosts on standard ports!